pkgs/tools/networking/teavpn2/nix.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/src/teavpn2/net/linux/iface.c b/src/teavpn2/net/linux/iface.c
+index a77c1c8..815b886 100644
+--- a/src/teavpn2/net/linux/iface.c
++++ b/src/teavpn2/net/linux/iface.c
+@@ -327,7 +327,8 @@ static __cold __always_inline const char *find_ip_cmd(void)
+ 		"/usr/sbin/ip",
+ 		"/usr/local/bin/ip",
+ 		"/usr/local/sbin/ip",
+-		"/data/data/com.termux/files/usr/bin/ip"
++	    "/data/data/com.termux/files/usr/bin/ip",
++       "/run/current-system/sw/bin/ip"
```

Comment:
You should probably use tab indentation here like the original file.

Reply:
done